docs/UserStories.md

```diff
@@ -0,0 +1,87 @@
+~~As a player, I want to play Ludo with the official rules.~~
```

Comment:
Why is this strikethrough and not just deleted? Is there any particular reason to keep this?